subprojects/packagefiles/theora/meson.build

```diff
@@ -0,0 +1,269 @@
+project(
+  'theora',
+  'c',
+  'cpp',
+  version: '1.1.1',
+  meson_version: '>= 0.59.0',
+  license: 'BSD-3-Clause',
+  default_options: ['c_std=c11'],
+)
+
+conf = configuration_data()
+
+enable_valgrind = get_option('valgrind-testing').allowed()
+
+test_env = []
+
+if enable_valgrind
+  valgrind_exe = find_program('valgrind')
```

Comment:
The usual point of a feature option is to do:

```
valgrind_exe = find_program('valgrind', required: get_option('valgrind-testing'))
```

The current state of this instead causes `-Dvalgrind-testing=auto` to force it to enabled...